Assets/TinyMCE/JoplinLists/src/main/ts/core/Selection.ts

```diff
@@ -49,7 +48,7 @@ const findParentListItemsNodes = function (editor, elms) {
     return parentLi ? parentLi : elm;
   });
 
-  return DomQuery.unique(listItemsElms);
+  return [...new Set(listItemsElms)];
```

Comment:
`DomQuery` and `editor.settings` were both removed from TinyMCE, and so can't be used by the lists plugin.

Since the JoplinLists plugin [doesn't build](https://github.com/laurent22/joplin/blob/5c711df2e42dc16e8819f76346be9b0d7e3ebbf3/Assets/TinyMCE/JoplinLists/README.md#L4), in addition to changing the source files, the built `lists.js` has been manually modified to also include these changes.